tests/common/factories/user.py

```diff
@@ -39,7 +39,7 @@ class Meta:
     class Params:
         inactive = factory.Trait(activation=factory.SubFactory(Activation))
         # A count that's appended to non-unique usernames to make them unique.
-        count = factory.Sequence(lambda n: "%d" % n)
+        count = factory.Sequence(lambda n: f"{n}")
```

Comment:
Is this doing the same thing? The previous one was casting it as an int I think. Is this something like `str(int(n))`?